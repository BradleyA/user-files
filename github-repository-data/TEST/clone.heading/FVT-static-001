#!/bin/bash
#	github-repository-data/TEST/clone.heading/FVT-static-001  3.239.459  2019-08-10T17:38:58.209821-05:00 (CDT)  https://github.com/BradleyA/user-files.git  uadmin  one-rpi3b.cptx86.com 3.238-116-g8bee949  
#	   github-repository-data/TEST/clone.heading/FVT-static-001 setup test case for file content 
### production standard 3.0 shellcheck
### production standard 5.1.160 Copyright
#       Copyright (c) 2019 Bradley Allen
#       MIT License is in the online DOCUMENTATION, DOCUMENTATION URL defined below.
BOLD=$(tput -Txterm bold)
NORMAL=$(tput -Txterm sgr0)
###

cat "${1}" > "${0}".out

#
diff "${0}".expected "${0}".out >/dev/null 2>&1
RETURN_CODE=${?}
if [ ${RETURN_CODE} -eq 0 ] ; then
	echo "${BOLD}Test case --->${NORMAL} ${0} ${1} ${RETURN_CODE} - No difference with expected output - ${BOLD}PASS - PASS${NORMAL}"
elif [ ${RETURN_CODE} -eq 1 ] ; then
	echo "${BOLD}Test case --->${NORMAL} ${0} ${1} ${RETURN_CODE} - Differences with expected output - ${BOLD}FAIL - FAIL${NORMAL}"
	diff -y "${0}".expected "${0}".out
else 
	echo "${BOLD}Test case --->${NORMAL} ${0} ${1} ${RETURN_CODE} - Test script ERROR - ${BOLD}FAIL - FAIL${NORMAL}"
fi
