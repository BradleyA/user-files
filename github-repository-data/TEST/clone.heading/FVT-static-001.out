Dates:
#	github-repository-data/TEST/clone.heading/FVT-static-001.out  3.241.463  2019-08-10T18:57:20.836730-05:00 (CDT)  https://github.com/BradleyA/user-files.git  uadmin  one-rpi3b.cptx86.com 3.240-1-g4514dd3  #
#	   hooks/post-commit remove FVT-*.out files run FVT-setup.sh if found in test case directory #
|:---
Clones:
Unique clones:
